packages/core/src/operator/operator-runtime/operatorState.ts

```diff
@@ -0,0 +1,38 @@
+import { ethers } from "ethers";
+import { NodeLicenseStatusMap, PublicNodeBucketInformation } from "../operatorRuntime.js";
+import { Challenge } from "../../index.js";
+import { SentryKey } from "@sentry/sentry-subgraph-client";
+
+/**
+ * Operator state cache type
+ */
+type OperatorState = {
+    cachedSigner: ethers.Signer;
+    cachedLogger: (log: string) => void;
+    safeStatusCallback: () => void;
+    onAssertionMissMatchCb: (publicNodeData: PublicNodeBucketInformation | undefined, challenge: Challenge, message: string) => void;
+    cachedBoostFactor: { [ownerAddress: string]: bigint };
+    operatorAddress: string;
+    cachedOperatorWallets: string[];
+    mintTimestamps: { [nodeLicenseId: string]: bigint };
+    cachedKeysOfOwner: { [keyId: string]: SentryKey };
+    nodeLicenseStatusMap: NodeLicenseStatusMap;
+    passedInOwnersAndPools: string[] | undefined;
```

Comment:
Possibly a comment on why/when/where passedInOwnersAndPools would be used/relevant?

Reply:
Its the "whitelist" of pools / owners the operator selects before starting the operatorRuntime, either in the cli or the desktop app.
This is so you don't pay gas for my keys just because I approved you to operate my keys / my pool.

Let me know if that makes sense.